packages/edit-site/src/components/page-pages/index.js

```diff
@@ -332,6 +332,7 @@ export default function PagePages() {
 				enableSorting: false,
 				filterBy: {
 					operators: [ OPERATOR_IN ],
+					isPrimary: true,
```

Comment:
This needs to be removed before merging the PR. It's set for demo purposes (simulate a situation where there is primary and secondary, so we can holistically review how the API works).

Reply:
This wasn't removed before merging the PR: https://github.com/WordPress/gutenberg/pull/58682